libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -695,6 +709,69 @@ bool AP_AHRS_NavEKF::get_relative_position_NED(Vector3f &vec) const
     }
 }
 
+// return a relative ground position in meters/second, North/East
```

Comment:
Inaccurate comment.
1. Writes relative ground position in meters, not meters/second
2. Does not return the position, it returns a bool representing the validity of the position, as stated in the 2nd comment line.